docs/api.rst

```diff
@@ -3290,6 +3302,12 @@ of :class:`enum.Enum`.
 
         The rule will timeout a user.
 
+    .. attribute:: block_member_interactions
+
+        Similar to :attr:`timeout` the rule will timeout a user except of having a duration.
+        This will request the user to edit it's profile.
```

Comment:
"it" is usually used to refer to objects rather than people, so I don't think it's the right fit here. It would probably be better to go with "their" instead?

(and just for the record, "it's" is the incorrect word to use here. "it's" is a shortening of "it is", whereas "its" is the possessive term, and refers to ownership of something)

Reply:
Ah okay, thanks for clarifying. As you may have already noticed, English is not my native language. 😄